### DISCUSSION TITLE
How to share dashboard in present mode?

### DISCUSSION THREAD

**Discussion by eudson** [atoti/atoti#551] [Q&A]
Hi,

I am new to Atoti, have been using it for a couple of weeks now and I really like it.

I work for a non-for-profit organization and we have a project where I would like to implement the analysis and disseminate the results using Atoti, I managed to deploy it on a server using the notebooks but I see that one can still switch from the `Present` mode and use the `Content Editor`. How can I hide the content editor?

Thank you

**Comment by bloussou** [comment]
Hello,
Indeed, in the community edition of atoti, saved dashboards can be edited by anyone able to access them. There is no way to force users to stay in the Present mode.

With Atoti+ however, you can configure security, [create users](https://docs.atoti.io/0.6.5/lib/atoti-plus/atoti_plus.security.html#module-atoti_plus.security) and grant them the `ROLE_SHARE` role to give them access to the [**Share popup**](https://activeviam.com/activeui/documentation/5.0.10/docs/user-guides/exporting-and-sharing#sharing-dashboards). Users with whom you've shared the dashboard with Reader only permission [will then see this dashboard in **Present** mode by default](https://activeviam.com/activeui/documentation/5.0.10/docs/user-guides/exporting-and-sharing#viewing-a-dashboard-with-read-only-permission).

You can reach out to [support@atoti.io](mailto:support@atoti.io) if you would like to try Atoti+.
